test/integration/serverless/serverless.yml

```diff
@@ -582,6 +582,7 @@ functions:
     environment:
       AWS_LAMBDA_EXEC_WRAPPER: /opt/otel-instrument
       DD_API_KEY: NO_NEED_TO_BE_VALID
+      DD_APM_FEATURES: disable_operation_and_resource_name_logic_v2
```

Comment:
Maybe instead of adding this env var, we should update the test snapshots. Since customers aren't super likely to use this env var, that would allow our tests to mirror customer environments more closely. Not a big deal, but something to consider for next time we're in here making changes.